server/src/interfaces.ts

```diff
@@ -61,26 +61,42 @@ export interface CommitComputedProperties {
 
 /**
  * CommitRequest Interface that contains the fields that will be provided
- * by the client in the POST/PUT request body.
+ * by the client in the request body to create a Commit.
  */
 export interface CommitRequest {
   customerId: number;
   listingId: number;
 }
 
+/**
+ * CommitPaymentRequest Interface that contains the fields that will be provided
+ * by the client in the request body.
+ */
+export interface CommitPaymentRequest {
```

Comment:
@ifhuang Oh I was thinking we shouldnt use that because they might not want to send items that they purchase to the same address everytime. 
In the future, #167 will be implemented and the default delivery details will be auto-filled for the user on the frontend

Reply:
I was thinking it's annoying that customer need to input address every time. But your #167 (set default and auto-fill) should solve that.